docs/eventing/broker/README.md

```diff
@@ -1,111 +1,33 @@
-  name: default
-spec:
-  # Configuration specific to this broker.
-  config:
-    apiVersion: v1
-    kind: ConfigMap
-    name: config-br-default-channel
-    namespace: knative-eventing
-```
+Before you can use the Knative Broker, you must install a channel provider, such as InMemoryChannel, Kafka or Nats.
 
-More complex example, showing the same `Broker` as above
-but with failed events being delivered to Knative Service called `dlq-service`
+**NOTE:** InMemoryChannel channels are for development use only and must not be used in a production deployment.
 
-```yaml
-apiVersion: eventing.knative.dev/v1
-kind: Broker
-metadata:
-  annotations:
-    eventing.knative.dev/broker.class: MTChannelBasedBroker
-  name: default
-spec:
-  # Configuration specific to this broker.
-  config:
-    apiVersion: v1
-    kind: ConfigMap
-    name: config-br-default-channel
-    namespace: knative-eventing
-  # Where to deliver Events that failed to be processed.
-  delivery:
-    deadLetterSink:
-      ref:
-        apiVersion: serving.knative.dev/v1
-        kind: Service
-        name: dlq-service
-```
+For more information on which channels are available and how to install them,
+see the list of [available channels](https://knative.dev/docs/eventing/channels/channels-crds/).
 
-### Creating a broker using defaults
+## How it works
+<!--TODO: Add a diagram that shows this-->
+When an Event is sent to the Broker, all metadata other than the `CloudEvent` data is stripped away.
+Unless the information existed as a `CloudEvent` attribute, no information is retained about how this Event entered the Broker.
 
-Knative Eventing provides a `ConfigMap` which by default lives in
-`knative-eventing` namespace and is called `config-br-defaults`. Out of the box
-it comes configured to create
-[MT Channel Based Brokers](./mt-channel-based-broker.md) as well as In Memory
-Channels. If you are using a different Broker implementation, or want to use
-a different channel implementation, you should modify the `ConfigMap`
-accordingly. You can read more details on how to use
-[config-br-defaults `ConfigMap`](./config-br-defaults.md)
+Once an Event has entered the Broker, it can be forwarded to event Channels by using Subscrbers and Triggers.
+This event delivery mechanism hides details of event routing from the event producer and event consumer.
 
-```yaml
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: config-br-defaults
-  namespace: knative-eventing
-  labels:
-    eventing.knative.dev/release: v0.16.0
-data:
-  # Configuration for defaulting channels that do not specify CRD implementations.
-  default-br-config: |
-    clusterDefault:
-      brokerClass: MTChannelBasedBroker
-      apiVersion: v1
-      kind: ConfigMap
-      name: config-br-default-channel
-      namespace: knative-eventing
-```
+Subscribers register the interest of an event sink in a particular class of events, so that the sink will receive the type of events specified.
```

Comment:
There is not really a concept of Subscriber separate from Trigger (though if that concept is useful in docs we could define it). Is it a user persona or a field in Trigger? If it's the field, then I suggest this wording:

Triggers register a subscriber's interest in a particular class of events, so that the subscriber's event sink will receive events that match the Trigger's filter.